activities/organization/finance-updates/finance-update-december-2020.md

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
This folder structure is incorrect: `` activities/organization/finance-updates/``
It creates a new ``organization`` folder within ``activities``.
It should be ``/organization/finance-updates/``.

Reply:
and this is what threw me off, I'll correct it